internal/dao/permission.go

```diff
@@ -113,6 +113,17 @@ func GetAccountProjectsByPermission(ctx context.Context, db bun.IDB, accountID,
 	return aps, err
 }
 
+func GetAccountProjectByPermission(ctx context.Context, db bun.IDB, accountID uuid.UUID, permission string) (models.AccountPermission, error) {
+	var aps models.AccountPermission
+
+	err := db.NewSelect().Model(&aps).
+		Where("account_id = ?", accountID).
+		Where("permission_name = ?", permission).
```

Comment:
Shouldn't we consider `partner_id` and `organization_id` when fetching account details?

Reply:
at this point, we have account information from user session. Also until one of the recent release identities does not have partner, organization information - for backward compatibility, we can relay on the logged in user session information.

Reply:
looking into a fix which would consider organization, partner information if available otherwise ignore to ensure backward compatibility.